content/uk/docs/reference/glossary/pod-security-policy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Політики безпеки Podʼа
+id: pod-security-policy
+date: 2018-04-12
+full_link: /docs/concepts/security/pod-security-policy/
+short_description: >
+  Дозволяєють деталізовану авторизацію створення та оновлення Podʼів.
+
+aka: 
+- Pod Security Policy
```

Comment:
```suggestion
- PodSecurityPolicy
```
Is this better?

Reply:
"Pod Security Policy" is the original name of the glossary item. I added it to the aka for those who want to find it as well.
<img width="2794" alt="" src="https://github.com/kubernetes/website/assets/369696/e2a590ec-aa8f-48e7-b8f9-df90eafafe2d">

Reply:
I prefer to leave it as it is, if you don't mind.